ship.config.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable import/no-commonjs */
+const fs = require('fs');
+const path = require('path');
+
+const packages = [
+  'packages/react-instantsearch-core',
+  'packages/react-instantsearch-dom-maps',
+  'packages/react-instantsearch-dom',
+  'packages/react-instantsearch-native',
+  'packages/react-instantsearch',
+];
+
+module.exports = {
+  monorepo: {
+    mainVersionFile: 'lerna.json',
+    packagesToBump: packages,
+    packagesToPublish: packages,
+  },
+  versionUpdated: ({ version, exec, dir }) => {
+    // write to `packages/react-instantsearch-core/src/core/version.js`
+    fs.writeFileSync(
+      path.resolve(
+        dir,
+        'packages',
+        'react-instantsearch-core',
+        'src',
+        'core',
+        'version.js'
+      ),
+      `export default '${version}';\n`
+    );
+
+    // update version in top level package
+    exec(`mversion ${version}`);
+
+    // update version in packages & dependencies
+    exec(`lerna version ${version} --no-git-tag-version --no-push --yes`);
+  },
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType;
+    if (releaseType === 'patch' && fix === 0) {
+      return false;
+    }
+    return true;
+  },
+  pullRequestTeamReviewers: ['instantsearch-for-websites'],
+  buildCommand: ({ version }) =>
+    `NODE_ENV=production VERSION=${version} yarn build`,
+  slack: {
+    // disable slack notification for `prepared` lifecycle.
+    // Ship.js will send slack message only for `releaseSuccess`.
+    prepared: null,
+    releaseSuccess: ({
+      appName,
+      version,
+      tagName,
+      latestCommitHash,
+      latestCommitUrl,
+      repoURL,
+    }) => ({
+      pretext: [
+        `:tada: Successfully released *${appName}@${version}*`,
+        '',
+        `Make sure to run \`yarn run update_examples ${version}\` in \`react-instantsearch\`.`,
```

Comment:
can we wait until the packages have been published and run the script here?

Reply:
https://github.com/algolia/react-instantsearch/pull/3017/commits/77ce2543ce8a9b87c3b4f6f31b67c0d56bf992b9 
WDTY?